app/build.gradle

```diff
@@ -1,14 +1,15 @@
 apply plugin: 'com.android.application'
 
 android {
-  compileSdkVersion 28
-
+  compileSdkVersion 29
+  buildToolsVersion "29.0.2"
```

Comment:
This field is not a required field. You can delete the `buildToolsVersion` line. This is automatically picked up from the  one installed on the machine.